app/templates/all_questions/cy/cof_r3w3_all_questions_cy.html

```diff
@@ -0,0 +1,1634 @@
+{% extends "base.html" %}
+{%- from 'govuk_frontend_jinja/components/inset-text/macro.html' import govukInsetText -%}
+{%- from "govuk_frontend_jinja/components/button/macro.html" import govukButton -%}
+
+{% from "partials/file-formats.html" import file_formats %}
+{% set pageHeading %}{% trans %}Full list of application questions{% endtrans %}{% endset %}
+{% block content %}
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        <span class="govuk-caption-l">{% trans %}{{fund_title}}{% endtrans %} {% trans %}{{round_title}}{% endtrans %}
+        </span>
+        <h1 class="govuk-heading-xl">{{pageHeading}}</h1>
+<div class="govuk-!-margin-bottom-8">
+  <h2 class="govuk-heading-m ">
+    {% trans %}Table of contents{% endtrans %}
+  </h2>
+  <ol class="govuk-list govuk-list--number">
+    <li>
+      <a class="govuk-link" href="#ynglŷn-â'ch-sefydliad">
+        Ynglŷn â'ch sefydliad
+      </a>
+    </li>
+    <li>
+      <a class="govuk-link" href="#ynglŷn-â'ch-prosiect">
+        Ynglŷn â'ch prosiect
+      </a>
+    </li>
+    <li>
+      <a class="govuk-link" href="#achos-strategol">
+        Achos strategol
+      </a>
+    </li>
+    <li>
+      <a class="govuk-link" href="#achos-rheoli">
+        Achos rheoli
+      </a>
+    </li>
+    <li>
+      <a class="govuk-link" href="#gwirio-datganiadau">
+        Gwirio datganiadau
+      </a>
+    </li>
+  </ol>
+  <hr class="govuk-section-break govuk-section-break--l govuk-section-break--visible" />
+  <h2 class="govuk-heading-l" id="ynglŷn-â'ch-sefydliad">
+    1. Ynglŷn â'ch sefydliad
+  </h2>
+  <h3 class="govuk-heading-m">
+    1.1. Gwybodaeth am y sefydliad
+  </h3>
+  <h4 class="govuk-heading-s">
+    1.1.1. Enwau'r sefydliad
+  </h4>
+  <p class="govuk-body">
+    Eich rhif traciwr unigryw
+  </p>
+  <p class="govuk-body">
+    Roedd hwn wedi'i gynnwys yn y llythyr yr anfonom i gadarnhau eich datganiad o ddiddordeb llwyddiannus.
+  </p>
+  <p class="govuk-body">
+    Er enghraifft, â€˜ANON-###-###-###â€™
```

Comment:
Why are there special characters? shouldn't it just be "ANON-###-###-###" like we got for English?